test/fulfillmentSpec.js

```diff
@@ -342,9 +409,11 @@ describe('PUT /fulfillment', function () {
     yield this.request()
       .put(transfer.id + '/fulfillment')
       .send(this.cancellationConditionFulfillment)
-      .expect(200)
-      .expect(this.cancellationConditionFulfillment)
-      .expect(validator.validateFulfillment)
+      .expect(422)
+      .expect({
+        id: 'AlreadyRolledBackError',
+        message: 'This transfer has already been rejected'
+      })
```

Comment:
Ok. I was going off of [the spec](https://github.com/interledger/rfcs/blob/master/0004-ledger-plugin-interface/0004-ledger-plugin-interface.md#fulfillcondition) (see `AlreadyRolledBackError`).